de.fu_berlin.inf.dpp.intellij/.idea/scopes/scope_settings.xml

```diff
@@ -1,5 +0,0 @@
-<component name="DependencyValidationManager">
```

Comment:
Is there a reason to remove this file?

Reply:
This config is only used for creating scopes which are used as alternative view of the project sources. The current config is useless